libs/components/data-manager/src/lib/modules/data-manager/data-manager-toolbar/data-manager-toolbar.component.html

```
@@ -13,7 +15,7 @@
           </sky-toolbar-item>
         }
 
-        @if (activeView?.sortEnabled) {
+        @if (activeView?.sortEnabled || sortOptions().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)